packages/launchpad-navigator/src/floating-navigator/index.tsx

```diff
@@ -9,7 +11,11 @@ export type FloatingNavigatorProps = {
 
 const FloatingNavigator = ( { siteSlug }: FloatingNavigatorProps ) => {
 	const launchpadContext = 'launchpad-navigator';
-	const checklistSlug = 'intent-build';
+	const checklistSlug = select( LaunchpadNavigator.store ).getActiveChecklistSlug() || null;
+
+	if ( ! checklistSlug ) {
+		return null;
+	}
```

Comment:
Given that we're still adding basic chrome and content in #82601, I think we should also handle this case in that PR, mostly to say something like "You don't have a task list that you're working on."